addons/stock/models/stock_move.py

```diff
@@ -198,7 +198,7 @@ def _compute_display_assign_serial(self):
                 not move.origin_returned_move_id.id and
                 move.state not in ('done', 'cancel')
             )
-            move.display_assign_serial = move.has_tracking == 'serial' and move.display_import_lot
+            move.display_assign_serial = move.display_import_lot
```

Comment:
Check if the override of `_compute_display_assign_serial` in `mrp_subcontracting` doesn't need to be rework (it skips move not tracked by serial, but I guess it should not skip move tracked by lot now)